.gitignore

```diff
@@ -1,2 +1,3 @@
 *.pyc
 /.mypy_cache
+.hypothesis
```

Comment:
in general you shouldn't modify `.gitignore` files for repositories you don't own -- you can always configure a global `.gitignore` such that parts of your development environment don't leak into repositories you contribute to

for example, we don't use hypothesis at all in pre-commit/pre-commit-hooks so it doesn't make any sense to add a .gitignore entry for it

also generally you'd want to sort this

Reply:
ah, thanks for the pointer!